pkg/model/stack_serializer.go

```diff
@@ -371,6 +371,21 @@ func getAccessiblePorts(ports []PortRaw) []PortRaw {
 	return accessiblePorts
 }
 
+// setLivenessFromDependsOnServiceHealthy sets the liveness probe to true if the service depends on another service being healthy
+func setLivenessFromDependsOnServiceHealthy(svc *Service, svcName string, s *Stack) {
+	// loop for each service in the stack to find if some of it is dependent of svcName for being healthy
+	for _, depService := range s.Services {
+		// if dependent service has a service_healty dependency for svcName, then set liveness probe to true
+		if condition, ok := depService.DependsOn[svcName]; ok && condition.Condition == DependsOnServiceHealthy {
+			if svc.Healtcheck == nil {
```

Comment:
NIT: I just noticed that there is a typo on `svc.Healtcheck`. Should we change it to `svc.Healthcheck`? I see the yaml is correctly spelled